modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -1285,9 +1285,11 @@ Included when xref:configuration/configuration-settings.adoc#config_db.logs.quer
 | annotationData
 | Metadata attached to the transaction.
 
+[role=label--deprecated-2025.05]
```

Comment:
Cannot see this in the preview - maybe it needs to be done in a different way

Reply:
yeah, true. If you need to label something in a table, you should use common label syntax.